### PR TITLE
Updates and changes from feedback.

### PR DESCRIPTION
**Pull request by t-brown**
- Updated the agenda to reflect current changes.
- Included changes from feedback by testers.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/02-aws-getting-started/06-start-aws-cli.md [hunk 1]
```diff
@@ -52,13 +52,22 @@ source ~/.bash_profile
...
 ```
 
-5. Identify the AWS region with the following commands in the Cloud9 terminal:
+5. Install yq utility (you will need this utiity to work with the cluster configuration file, as it is in YAML format)
+
+```bash
+VERSION=v4.28.1
+BINARY=yq_linux_amd64
+mkdir -p $HOME/.local/bin
+wget https://github.com/mikefarah/yq/releases/download/${VERSION}/${BINARY} -O $HOME/.local/bin/yq && chmod +x $HOME/.local/bin/yq
```
Comment (kraman-aws) [inline content/02-aws-getting-started/06-start-aws-cli.md:67]: thanks for adding this. i assume you confirmed it picks the right version and does not need to be added in PATH?

File: content/03-hpc-aws-parallelcluster-workshop/09-connect-cluster.md [hunk 2]
```diff
@@ -13,37 +13,39 @@ When the **pcluster create-cluster** command is executed, AWS ParallelCluster ge
...
+Once the cluster goes into **CREATE COMPLETE**, you can connect to the head node via SSM.
 
-**DCV** is a full graphical remote desktop that allows you to run GUI applications on the head node. It doesn't require AWS account access but does require you to be able to connect to the head node on port **8443**.
+## SSM Connect
 
-## DCV Connect
+You can skip this step or you can read below if you are curious about the other ways to connect your cluster.
 
-1. Click on the **DCV** Button to connect:
+In addition to DCV, you can connect to your cluster via SSH using SSM.
```
Comment (kraman-aws) [inline content/03-hpc-aws-parallelcluster-workshop/09-connect-cluster.md:22]: above says you connect via SSM, here it says DCV.

File: content/03-hpc-aws-parallelcluster-workshop/09-connect-cluster.md [hunk 3]
```diff
@@ -13,37 +13,39 @@ When the **pcluster create-cluster** command is executed, AWS ParallelCluster ge
...
-**SSM Connect** It doesn't require any ports to be open on the head node, however it does require you to authenticate with the AWS account the instance it running in.
+![DCV Connect](/images/sc22/PCM_DCV.png)
 
-1. Click on the **Shell** Button to connect:
+2. As a one-time step since DCV uses self-signed certificates you'll need to click on **Advanced** > **Proceed to Unsafe**:
 
-![SSM Connect](/images/sc22/PCM_Shell.png)
+![Browser Warning](/images/sc22/browser-warning.png)
+
+3. Next to launch a terminal (where the rest of the lab will run) we'll click **Activities** > **Terminal**:
```
Comment (kraman-aws) [inline content/03-hpc-aws-parallelcluster-workshop/09-connect-cluster.md:47]: is this via DCV or via SSM?

File: content/03-hpc-aws-parallelcluster-workshop/14-exit-cluster.md [hunk 4]
```diff
@@ -14,6 +14,12 @@ In this lab, you have:
 
 In the next lab, you will learn about [Amazon FSx for Lustre](https://aws.amazon.com/fsx/lustre/).
 
+We need to stop the compute cluster, so we can update it in the next lab. In your AWS Cloud9 terminal window paste the following command
```
Comment (kraman-aws) [inline content/03-hpc-aws-parallelcluster-workshop/14-exit-cluster.md:17]: We need to stop the compute fleet, so ..

**Review by kraman-aws — CHANGES_REQUESTED** [review]
added a few comments, rest looks good to me. can you make sure that this PR gets merged  https://github.com/aws-samples/aws-hpc-tutorials/pull/332 first before merging this

File: content/02-aws-getting-started/06-start-aws-cli.md (hunk 1 above)
Comment (t-brown) [reply content/02-aws-getting-started/06-start-aws-cli.md:67]: Yes, `~/.local/bin` is already in the users path.

**Review by mhuguesaws — COMMENTED** [review]
Please address conflicts.

**Review by kraman-aws — APPROVED** [review]

**Pull request merged**